compile/enum.go

```diff
@@ -27,15 +27,17 @@ import (
 
 // EnumSpec represents an enum defined in the Thrift file.
 type EnumSpec struct {
-	Name  string
-	File  string
-	Items []EnumItem
+	Name        string
+	File        string
+	Items       []EnumItem
+	Annotations []*AnnotationSpec
```

Comment:
If you add `Annotations()` to the TypeSpec interface, you'll probably want to rename this to `Annots`.